### PR TITLE
Login implementation initial phase

### PR DESCRIPTION
**Pull request by lakhina**
-  Add seperate route for login form and request
-  Add Auth model
-  UserCommand instance variable update
-  Implement user authentication call

### CODE REVIEW

**Comment by lakhina** [comment]
@ignasi35 @TimMoore @jroper  Please review the initial implementation of Login Service

File: user-impl/src/main/java/com/example/auction/user/impl/UserEntity.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ private Behavior notCreated() {
         );
 
         b.setCommandHandler(CreateUser.class, (create, ctx) -> {
-            User user = new User(UUID.fromString(entityId()), create.getName());
+            User user = new User(UUID.fromString(entityId()), create.getName(),"admin","admin");
```
Comment (jroper) [inline user-impl/src/main/java/com/example/auction/user/impl/UserEntity.java]: What I would do here is have the username as a property of user, but have the password as its own event, so you'd persist two events here, a `UserCreated` event, followed by a `PasswordUpdated` event.

In addition the `PasswordUpdated` event should store the password as a hash.  Bcrypt is good for hashing passwords:

http://www.mindrot.org/projects/jBCrypt/

Comment (lakhina) [inline user-impl/src/main/java/com/example/auction/user/impl/UserEntity.java]: @jroper I will persist this as two seperate events `UserCreated` and `PasswordUpdated` in my new commits.

Comment (lakhina) [inline user-impl/src/main/java/com/example/auction/user/impl/UserEntity.java]: @jroper Do you to store password in a separate table or in the same table you want to store the encrypted password?

**Comment by ignasi35** [comment]
Hi @lakhina did you close this by mistake? Are you moving your work to #95  ?

**Comment by lakhina** [comment]
@ignasi35 I am moving work to #95 as I had to use new branch. Reimplementing the whole work with new approach,

**Pull request closed without merging**